### PR TITLE
Apps 9798 add publish job

### PR DESCRIPTION
**Pull request by AlexeyCL**

### CODE REVIEW

File: .github/workflows/publish-package.yml [hunk 1]
```diff
@@ -40,5 +34,36 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: actions/checkout@v2
```
Comment (FelixZilber) [inline .github/workflows/publish-package.yml:37]: line 15, better to use actions/checkout@v2 as well

File: .github/workflows/publish-package.yml [hunk 2]
```diff
@@ -40,5 +34,36 @@ jobs:
...
+
+      - name: Install packages
+        run: yarn install
+
       - name: Release new package
-        run: standard-version --dry-run
+        run: |
+          git config user.name ${{ secrets.USERNAME }}
+          git config user.email "${{ secrets.MAIL }}"
+          yarn release
```
Comment (FelixZilber) [inline .github/workflows/publish-package.yml:48]: need a better name for this phase. it's not releasing a package at this point

File: .github/workflows/publish-package.yml [hunk 3]
```diff
@@ -40,5 +34,36 @@ jobs:
...
+
+      - name: push changes
+        run: git push --follow-tags origin HEAD
```
Comment (FelixZilber) [inline .github/workflows/publish-package.yml:51]: use master instead of HEAD

File: .github/workflows/publish-package.yml [hunk 4]
```diff
@@ -40,5 +34,36 @@ jobs:
...
+
+      - uses: JS-DevTools/npm-publish@v1
+        with:
+          token: ${{ secrets.NPM_TOKEN }}
+
+  slack_notification:
+    needs: [ release_package ]
```
Comment (FelixZilber) [inline .github/workflows/publish-package.yml:58]: this will be executed only in case of success. need to send a message also in case of failure

**Review by FelixZilber — APPROVED** [review]

File: .github/workflows/publish-package.yml (hunk 3 above)
Comment (AlexeyCL) [reply .github/workflows/publish-package.yml:51]: done

File: .github/workflows/publish-package.yml (hunk 4 above)
Comment (AlexeyCL) [reply .github/workflows/publish-package.yml:58]: done

File: .github/workflows/publish-package.yml (hunk 2 above)
Comment (AlexeyCL) [reply .github/workflows/publish-package.yml:48]: done

File: .github/workflows/publish-package.yml (hunk 1 above)
Comment (AlexeyCL) [reply .github/workflows/publish-package.yml:37]: done

**Review by refael-m — APPROVED** [review]

**Pull request merged**